rules/autoconsent/monzo-com.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "monzo.com",
+    "prehideSelectors": [".cookie-alert"],
+    "detectCmp": [
+        { 
+            "exists": ".cookie-alert"
+        },
+        {
+            "exists": "[role=\"dialog\"]"
+        }
+],
```

Comment:
this looks very generic. It says "there should be an element with cookie-alert AND there should be an element with role="dialog"". But I believe we need them on the SAME element, did you mean `.cookie-alert[role=\"dialog\"]`?

And even like this, we might need an extra check, given that we already had a conflict with another rule, I can easily imagine another page using .cookie-alert with role=dialog. Could you add another check as well? Perhaps something matching the inner structure of the banner?

Reply:
see comment below: https://github.com/duckduckgo/autoconsent/pull/22#issuecomment-1204897857